server/src/mindustry/server/ServerControl.java

```diff
@@ -1048,11 +1056,16 @@ public void handleCommandString(String line){
         }
     }
 
+    @Deprecated
     public void setNextMap(Map map){
-        nextMapOverride = map;
+        maps.setNextMapOverride(map);
+    }
+
+    public void play(Runnable run){
+        play(true, run);
     }
 
-    private void play(boolean wait, Runnable run){
+    public void play(boolean wait, Runnable run){
```

Comment:
undocumented parameters and method, explain what it does if you're going to make it public